collectors/charts.d.plugin/libreswan/README.md

```diff
@@ -1,6 +1,6 @@
-# libreswan
+# LibreSwan monitoring with Netdata
```

Comment:
`LibreSwan` is IPsec VPN implementation

> https://libreswan.org/

Modules monitors Libreswan IPsec Tunnels

Reply:
**Libreswan IPSec tunnel monitoring with Netdata**